test.sh

```diff
@@ -21,48 +25,135 @@ function prepareOutputDir() {
 }
 
 function createTestEnvironment() {
+  echo '-----------------------test.sh------------------------------'
   echo "Creating test environment"
-  echo '-----'
+  echo '------------------------------------------------------------'
 
   # Build test container & start the cluster
-  docker-compose pull postgres possum
-  docker-compose build --pull test
-  docker-compose up -d client possum postgres
+  docker-compose pull client conjur postgres
+  docker-compose build --pull test test-https conjur-proxy-nginx
+  docker-compose up -d client conjur postgres test-https
 
-  # Delay to allow time for Possum to come up
+  # Delay to allow time for conjur to come up
   # TODO: remove this once we have HEALTHCHECK in place
-  echo 'Waiting for possum to be healthy'
+  echo 'Waiting for conjur server to be healthy'
   docker-compose run --rm test ./wait_for_server.sh
 }
 
 function loadTestPolicy() {
+  echo '-----------------------test.sh------------------------------'
   echo "Loading test policy"
-  echo '-----'
+  echo '------------------------------------------------------------'
 
-  # get possum container id
-  possum_client_cid=$(docker-compose ps -q client)
+  # get conjur container id
+  conjur_client_cid=$(docker-compose ps -q client)
 
-  api_key=$(docker-compose exec -T possum rails r "print Credentials['cucumber:user:admin'].api_key")
+  api_key=$(docker-compose exec -T conjur rails r "print Credentials['cucumber:user:admin'].api_key")
```

Comment:
It would be great if we can get rid of these `docker exec`s, but maybe we need to fix that in cyberark/conjur.